.pre-commit-config.yaml

```diff
@@ -37,7 +37,7 @@ repos:
       - id: oxipng
         stages: [manual]
   - repo: https://github.com/pre-commit/mirrors-eslint
-    rev: v9.13.0
```

Comment:
Hi there, I think a change needed for the eslint hook is to config the files/types so when you do a commit locally this hook actually runs if you modify the `.jsx` files. From the hook README (https://github.com/pre-commit/mirrors-eslint?tab=readme-ov-file#using-eslint-with-pre-commit), adding to the config something like the following could help:

```yaml
        files: \.[jt]sx?$  # *.js, *.jsx, *.ts and *.tsx
        types: [file]
```